app/views/dashboard/index.html.erb

```diff
@@ -15,3 +15,13 @@
     <% end %>
   </ol>
 </section>
+<section id="items-ready-to-ship">
+  <h2>Items Ready to Ship</h2>
+  <ul>
+    <% @items_ready_to_ship.each do |invoice| %>
+      <% invoice.invoice_items.each do |invoice_item| %>
+        <li><%= invoice_item.item.name %> - Invoice#<%= link_to "#{invoice_item.invoice.id}", merchant_invoice_path(@merchant, invoice_item.invoice), method: :get %> </li>
```

Comment:
Great work tackling this nested iteration! Let's see if there's any way we can push more logic here off to the controller or model when we refactor to help clean up our view.